README.md

```diff
@@ -1,12 +1,16 @@
-# Deis Workflow v2
+# Deis Controller
 
 [![Build Status](https://travis-ci.org/deis/controller.svg?branch=master)](https://travis-ci.org/deis/controller)
 [![codecov.io](https://codecov.io/github/deis/controller/coverage.svg?branch=master)](https://codecov.io/github/deis/controller?branch=master)
 [![Docker Repository on Quay](https://quay.io/repository/deisci/controller/status "Docker Repository on Quay")](https://quay.io/repository/deisci/controller)
 
-Deis (pronounced DAY-iss) Workflow is an open source Platform as a Service (PaaS) that adds a developer-friendly layer to any [Kubernetes](http://kubernetes.io) cluster, making it easy to deploy and manage applications on your own servers.
+The Controller is the central API server for [Deis Workflow][workflow]. It is installed on a [Kubernetes](http://kubernetes.io) cluster, making it easy to deploy and manage applications on your own cluster. Below is a non-exhaustive list of things it can do:
 
-For more information about the Deis Workflow, please visit the main project page at https://github.com/deis/workflow.
+* Create a new application
+* Delete an application
+* Scale an application
+* Configure an application
+* Create a new user
```

Comment:
The "template" for our readmes has had the generic Deis introduction, and this added text in the "About" section below. Can we keep things consistent?